pkg/xfs/mount_linux.go

```diff
@@ -20,17 +20,36 @@ package xfs
 
 import (
 	"errors"
+	"fmt"
 	"os"
+	"path"
 
 	"github.com/minio/directpv/pkg/sys"
 )
 
 func mount(device, target string) error {
-	if err := os.Mkdir(target, 0o777); err != nil && !errors.Is(err, os.ErrExist) {
+	if err := sys.Mkdir(target, 0o777); err != nil && !errors.Is(err, os.ErrExist) {
 		return err
 	}
 
-	return sys.Mount(device, target, "xfs", []string{"noatime"}, "prjquota")
+	if err := sys.Mount(device, target, "xfs", []string{"noatime"}, "prjquota"); err != nil {
+		return err
+	}
+
+	name := path.Base(device)
+	if name == "/" || name == "." {
+		return fmt.Errorf("unable to get device name from device %v", device)
+	}
+
+	if err := os.WriteFile("/sys/fs/xfs/"+name+"/error/metadata/EIO/retry_timeout_seconds", []byte("60"), 0o644); err != nil {
+		return fmt.Errorf("unable to set EIO retry_timeout_seconds for device %v; %w", name, err)
```

Comment:
Actually even 30s is more than enough @balamurugana

Reply:
Also we need to set this to 1 for max_retries, we really don't want any retry via XFS beyond atleast 1 entry.

And perhaps we should reduce the retry only up-to 5s or worst 10s.